go/carmen/carmen_test.go

```diff
@@ -32,8 +32,14 @@ var testNonArchiveConfig = Configuration{
 	Archive: Archive(state.NoArchive),
 }
 
+var testProperties = Properties{
+	LiveDBCache:  "2000",
+	ArchiveCache: "2000",
+	StorageCache: "100",
+}
+
 func TestCarmen_DatabaseLiveCycle(t *testing.T) {
-	db, err := OpenDatabase(t.TempDir(), testConfig, nil)
+	db, err := OpenDatabase(t.TempDir(), testConfig, testProperties)
```

Comment:
nit: provide a `OpenTestDatabase(t)` function?

Reply:
ok, yes, introduced this method